release kaboomed

```
[INFO    ] Previous released version: v0.0.0
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.10.7/x64/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/opt/hostedtoolcache/Python/3.10.7/x64/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/home/runner/work/release-action/release-action/datalad_release_action/__main__.py", line 141, in <module>
    main()
  File "/home/runner/work/release-action/release-action/release/venv/lib/python3.10/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/home/runner/work/release-action/release-action/release/venv/lib/python3.10/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/home/runner/work/release-action/release-action/release/venv/lib/python3.10/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/runner/work/release-action/release-action/release/venv/lib/python3.10/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/runner/work/release-action/release-action/release/venv/lib/python3.10/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/home/runner/work/release-action/release-action/release/venv/lib/python3.10/site-packages/click/decorators.py", line 38, in new_func
    return f(get_current_context().obj, *args, **kwargs)
  File "/home/runner/work/release-action/release-action/datalad_release_action/__main__.py", line 120, in release
    next_version = bump_version(prev_version, bump)
  File "/home/runner/work/release-action/release-action/datalad_release_action/versions.py", line 72, in bump_version
    parts = [int(p) for p in v.split(".")]
  File "/home/runner/work/release-action/release-action/datalad_release_action/versions.py", line 72, in <listcomp>
    parts = [int(p) for p in v.split(".")]
ValueError: invalid literal for int() with base 10: 'v0'
Error: Process completed with exit code 1.
```

--- a/changelog.d/pr-19.md
+++ b/changelog.d/pr-19.md
@@ -1,0 +1,3 @@
+### 🐛 Bug Fixes
+
+- Fix handling of version tags that start with `v`.  Fixes [#18](https://github.com/datalad/release-action/issues/18) via [PR #19](https://github.com/datalad/release-action/pull/19) (by [@jwodder](https://github.com/jwodder))

--- a/datalad_release_action/versions.py
+++ b/datalad_release_action/versions.py
@@ -68,8 +68,10 @@ def bump_version(v: str, bump: Bump) -> str:
     '1.2.4'
     >>> bump_version("1.2.3.4", Bump.PATCH)
     '1.2.4.0'
+    >>> bump_version("v1.2.3", Bump.PATCH)
+    '1.2.4'
     """
-    parts = [int(p) for p in v.split(".")]
+    parts = [int(p) for p in v.lstrip("v").split(".")]
     vs = parts + [0] * (bump.level + 1 - len(parts))
     vs[bump.level] += 1
     vs[bump.level + 1 :] = [0] * len(vs[bump.level + 1 :])

